presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestRoles.java

```diff
@@ -561,6 +561,21 @@ public void testGrantAllRevokeSelect()
     }
 
     @Test(groups = {HIVE_CONNECTOR, ROLES, AUTHORIZATION, PROFILE_SPECIFIC_TESTS})
+    public void testGrantSelectRevokeAll()
+            throws Exception
+    {
+        final String tableName = "default.test_table";
+        onPresto().executeQuery("CREATE ROLE role1");
+        onPresto().executeQuery("GRANT role1 TO alice");
+        onPresto().executeQuery(format("CREATE TABLE %s (foo BIGINT)", tableName));
+        onPresto().executeQuery(format("GRANT SELECT ON %s TO role1", tableName));
```

Comment:
I think what you're missing is the `role` keyword. Without it, `role1` would be interpreted as the user name rather than a role name.